unittest/unit/find-missing.js

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const assert = require('chai').assert;
+
+const proxyquire = require('proxyquire');
+
+const tests = {
+  'api.AbortController': {
+    tests: [{code: '"AbortController" in self', prefix: ''}],
+    category: 'api',
+    resources: {},
+    exposure: ['Window', 'Worker', 'ServiceWorker']
+  },
+  'api.AbortController.signal': {
+    tests: [{code: '"AbortController" in self && "signal" in AbortController.prototype', prefix: ''}],
+    category: 'api',
+    resources: {
+      'audio-blip': {
+        type: 'audio',
+        src: '/media/blip.mp3'
+      }
+    },
+    exposure: ['Window', 'Worker']
+  },
+  'css.properties.font-family': {
+    tests: [{code: '"fontFamily" in document.body.style || CSS.supports("font-family", "inherit")', prefix: ''}],
+    category: 'css',
+    resources: {},
+    exposure: ['Window']
+  },
+  'javascript.builtins.array': {
+    tests: [{code: '[1, 2, 3]', prefix: ''}],
+    category: 'javascript',
+    resources: {},
+    exposure: ['JavaScript']
+  }
+};
+const bcd = {
+  api: {
+    AbortController: {
+      __compat: {},
+      dummy: {
+        __compat: {}
+      },
+      signal: {
+        __compat: {}
+      }
+    },
+    DummyAPI: {
+      __compat: {},
+      dummy: {
+        __compat: {}
+      }
+    }
+  },
+  css: {
+    properties: {
+      'font-family': {
+        __compat: {}
+      },
+      'font-face': {
+        __compat: {}
+      }
+    }
+  }
+};
+
+const {traverseFeatures, findMissing} = proxyquire('../../find-missing', {
+  './tests.json': Object.assign({}, tests),
```

Comment:
Why the `Object.assign` here? `tests` isn't used elsewhere, so it's not to make a copy and avoid interaction between tests. If it were, then a deep copy would be more appropriate.

Reply:
What's odd is that without the `Object.assign`, it seems to merge both the dummy object and the real JSON file, though I'm not sure why.  Wondering if it's a `proxyquire` bug...

Reply:
Found the issue: it looks like `proxyquire` has call-thru capabilities, so that anything that isn't mocked in `proxyquire` is obtained from the original dependency.  That's why BCD is being loaded, and in fact, JSON files as well in all other tests.

I opened #594 to resolve this.